simde/x86/avx2.h

```diff
@@ -1870,6 +1870,32 @@ simde_mm256_unpacklo_epi32 (simde__m256i a, simde__m256i b) {
 #  define _mm256_unpacklo_epi32(a, b) simde_mm256_unpacklo_epi32(a, b)
 #endif
 
+SIMDE__FUNCTION_ATTRIBUTES
+simde__m256i
+simde_mm256_unpacklo_epi64 (simde__m256i a, simde__m256i b) {
+#if defined(SIMDE_AVX2_NATIVE)
+  return _mm256_unpacklo_epi64(a, b);
+#else
+  simde__m256i_private
+    r_,
+    a_ = simde__m256i_to_private(a),
+    b_ = simde__m256i_to_private(b);
+
+#if defined(SIMDE__SHUFFLE_VECTOR)
+  r_.i64 =SIMDE__SHUFFLE_VECTOR(64, 32, a_.i64, b_.i64,
+                                1, 5, 3, 7);
```

Comment:
These values are wrong, which is why CI is failing.  Perhaps you forgot to add `-march=native` to your CFLAGS/CXXFLAGS, so only the fallbacks were running on your machine?  The easiest way to do that is to call cmake like `cmake .. -DCMAKE_{C,CXX}_FLAGS='-march=native'`.

You'll also need to regenerate the test cases, and make sure you copy the output from `/x86/avx2/mm256_unpacklo_epi64/native/c` or `/x86/avx2/mm256_unpacklo_epi64/native/cpp`, *not* `/x86/avx2/mm256_unpacklo_epi64/emul/c` or `/x86/avx2/mm256_unpacklo_epi64/emul/cpp`.  The former are the result of calling `_mm256_unpacklo_epi64`, the latter are the result of calling the portable version.

Also, this function call is short enough that it can go on one line.  That's really not a big deal; I'd normally probably just accept the patch, but since it needs some work anyways it would be nice if you could change that, too :)

Reply:
Yes, I the flags native flag was not check and I checked the didn't notice that the tests which were passing were emul and not native. Also I made a bug and used indexes of unpackhi instead of unpacklo. I will be more careful from next time.